about.html

```diff
@@ -0,0 +1,117 @@
+                        <li class="menu-li cc">CC Campaign</li>
+
+                    </ul>
+                </div>
+
+            </nav>
+        </div>
+
+    </header>
+    <main>
+        <section>
+            <div class="about-section">
+                <div class="intro">
+                    <h1 class="about-h1"> "CREATIVE COMMONS <br>GLOBAL SUMMIT ,<br> 2015"</h1>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eveniet voluptatum vero velit, ad
+                        incidunt ex, eligendi aliquid suscipit doloremque culpa nihil tempore placeat nostrum veniam
+                        similique aliquam rerum nisi neque atque. Unde quas iure et id illo necessitatibus debitis fuga
+                        expedita illum impedit voluptate veniam in molestias itaque tenetur quia ipsam, dignissimos
+                        ipsum, esse ratione eveniet obcaecati. A nemo ducimus mollitia tenetur officiis eveniet, dolores
+                        unde asperiores iste optio accusamus possimus laudantium qui! Nihil saepe totam numquam magnam
+                        aperiam quasi.</p>
+
+                    <h6>Please Contack us if you need any furthur information about this Bootcamp</h6>
+
+
+                </div>
+            </div>
+            <div class="about-our-logo">
+                <h2> CREATIVE COMMONS GLOBAL SUMMIT 2015 LOGO</h2>
+                <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus animi veritatis rem nesciunt
+                    temporibus, adipisci expedita voluptatibus culpa illo saepe et nisi suscipit quidem voluptatem!
+                    Tempora aliquam rem necessitatibus facilis.</p>
+                <div class="global-logo">
+                    <img src="./img/logo.png" alt="Logo" width="200px">
+                </div>
+
+
+
+        </section>
+        <section class="logo-section">
+            <div class="see-past">
+                <h3>SEE THE PAST CC GLOBAL SUMMITS</h3>
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Perspiciatis ut accusamus dignissimos sed
+                    deleniti natus laborum id voluptatibus atque voluptatum? Ipsa, ab. Placeat molestias pariatur
+                    officiis est earum. Blanditiis, aliquam.</p>
+                <div class="bootcamp">
+                    <div class="img1">
+                        <h4> 2013 <br> CREATIVE COMMONS GLOBAL SUMMIT</h4>
+                    </div>
+                    <div class="img2">
+
+                        <h4> 2014 <br> CREATIVE COMMONS GLOBAL SUMMIT</h4>
+                    </div>
+
+                </div>
+            </div>
+            </div>
+        </section>
+        <footer class="footer">
+            <div class="footer-block">
+                <img src="./img/logo.png" alt="Logo" width="200px">
+                <h3>@2015 Creative Commons Korea <br>Inc. All Rights Reserved.</h3>
+            </div>
+        </footer>
```

Comment:
[OPTIONAL] I think the footer of this page will look better if you can look for a logo that has a white background so that it can blend in with the black background on your footer because presently the footer is not really visible. 

| Your Design |Figma Design |
| :---:   | :---: | 
| <img width="843" alt="image" src="https://user-images.githubusercontent.com/73854999/203861319-b6da22ee-452c-47c1-bbef-f6b59df14ea6.png">|<img width="843" alt="image" src="https://user-images.githubusercontent.com/73854999/203861344-19178bb4-b529-4fc5-abfd-d601260829b5.png">|